pkgs/applications/science/misc/openmvs/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchFromGitHub, pkgconfig, cmake
+, eigen, opencv, ceres-solver, cgal, boost, vcg
+, gmp, mpfr, glog, google-gflags, libjpeg_turbo }:
+
+stdenv.mkDerivation rec {
+  version = "a3b3600";
+  name = "openmvs-${version}";
+
+  src = fetchFromGitHub {
+    owner = "cdcseacave";
+    repo = "openmvs";
+
+    rev = "a3b360016660a1397f6eb6c070c2c19bbb4c7590";
```

Comment:
@Mic92 , not really. It's more so because the release cycle has currently stagnated, and we would be using code from more than a year ago. In this time, significant amounts of commits and bug patches have been released that have significantly impacted the stability and robustness of the software.